app/src/store/provider.tsx

```diff
@@ -18,8 +17,8 @@ const StoreContext = React.createContext<Store | undefined>(undefined);
 export const StoreProvider: React.FC<{
   store: Store;
 }> = ({ children, store }) => {
-  const localStore = useLocalStore(() => store);
-  return <StoreContext.Provider value={localStore}>{children}</StoreContext.Provider>;
+  // const localStore = useLocalStore(() => store);
```

Comment:
nit: remove commented code?

Reply:
Grrrr. Not sure how I missed that one. Thanks

Reply:
This is fixed now.